spec/models/miq_request_workflow_spec.rb

```diff
@@ -434,6 +434,31 @@
     end
   end
 
+  context "#set_request_values" do
+    before do
+      workflow.set_request_values(values)
+    end
+    let(:values) { {:owner_email => owner.email} }
+    let(:owner)  { FactoryGirl.create(:user_with_email, :miq_groups => [FactoryGirl.create(:miq_group)]) }
+
+    it 'sets owner_group and requester_group' do
+      expect(values[:owner_group]).to eq(owner.current_group.description)
+      expect(values[:requester_group]).to eq(workflow.requester.miq_group_description)
+    end
+
+    it 'doesn\'t reset owner_group and requester_group on a second run' do
```

Comment:
Minor, but let's change this to `does not...` to avoid needing to escape character.  And it's just a better way of sayin' it.  😄